packages/react-portal-compat/README.md

```diff
@@ -1,5 +1,30 @@
 # @fluentui/react-portal-compat
 
-**React Portal Compat components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+Compat components for [React Portals](https://reactjs.org/docs/portals.html) between `@fluentui/react-components` & `@fluentui/react` or `@fluentui/react-northstar`.
 
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+With `PortalCompatProvider` components from `@fluentui/react-components` can be rendered properly in Portals created by `@fluentui/react` or `@fluentui/react-northstar`.
+
+### Installation
+
+> **NOTE:** this package is not being published yet
+
+```sh
+yarn add @fluentui/react-portal-compat
+```
+
+### Usage
+
+`PortalCompatProvider` should be an inner child of `FluentProvider`, no additional configuration is required for `@fluentui/react` or `@fluentui/react-northstar`.
```

Comment:
**thought:** can we add a dev time check to notify users if they are violating this rule ? ( can be done in separate PR )

Reply:
Added 👍